packages/fxa-admin-server/src/gql/account/account.resolver.spec.ts

```diff
@@ -308,4 +308,17 @@ describe('AccountResolver', () => {
     expect(result).toBeDefined();
     expect(result.length).toBe(1);
   });
+
+  it('edits locale', async () => {
+    const result1 = await resolver.editLocale(USER_1.uid, 'en-CA');
+    const result2 = await resolver.accountByEmail(USER_1.email, true, 'joe');
+    expect(result1).toBe(true);
+    expect(result2.locale).toBe('en-CA');
+  });
+
+  it('enforces length limit on locale', async () => {
+    await expect(async () => {
+      const result1 = await resolver.editLocale(USER_1.uid, 'x'.repeat(1000));
```

Comment:
nit: I don't see any code enforcing a limit; is this testing the size of the db column?  It doesn't seem necessary.

Reply:
It errors at the DB. I can remove the check.